### DISCUSSION TITLE
raising default verbosity level for debug builds

### DISCUSSION THREAD

**Discussion by farwyler** [helix-editor/helix#2698] [General] [answered]
When debugging (especially unexpectedly) i often wish i had cranked up the logging verbosity beforehand. 
Does it make sense to raise the default verbosity level for debug builds?
Also, raise to what? Although i generally expect more verbose output from debug builds, raising default verbosity to 2 (debug) might be way too spamy. In theory at least, actually most "spamy" log output is currently at info level. There are only a few cases where log::debug! is used.

https://github.com/helix-editor/helix/blob/9f8df05d96b0c475fefe079bca01c23eb492e6bb/helix-term/src/args.rs#L20
to
```rust
let mut args = Args {
  #[cfg(debug_assertions)]
  verbosity: 1,
  ..Args::default()
};
```

Any feelings regarding this?

**Comment by the-mikedavis** [comment, marked as answer]
The debug logs even at level 1 are pretty verbose. I prefer that verbosity is specified explicitly. Also, that patch would make it impossible to run a debug build without verbose logging.

Instead this can be solved with a shell alias:

```bash
alias checkhx='path/to/helix/target/debug/hx -v'
```
